test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -518,6 +519,9 @@ class PTLTestRunner(Plugin):
     def __init__(self):
         Plugin.__init__(self)
         self.param = None
+        self.repeat_tests = 1
+        self.test_repeat = 1
+        self.repeat_test_delay = 30
```

Comment:
I think 30 seconds delay is little bit high between tests run. Is there any reason behind this value ? It can be reduced to 10 seconds or even 5 seconds.